specification/vmware/resource-manager/readme.md

```diff
@@ -239,10 +254,43 @@ suppressions:
     # https://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options
     from: vmware.json
 
-  # just warnings
-  # - code: IgnoredPropertyNextToRef
-  # - code: LroLocationHeader
-  #   from: vmware.json
+  - code: XmsClientName
+    reason: false positives
+    from: vmware.json
+
+  - code: XmsClientNameParameter
+    reason: this warning is false postive
+    from: vmware.json
+
+  - code: docLinkLocale
+    reason: false positives
+    from: vmware.json
+
+  - code: PropertiesTypeObjectNoDefinition
+    reason: false positives
+    from: vmware.json
+
+  - code: LatestVersionOfCommonTypesMustBeUsed
+    reason: this warning is TODO
+    from: vmware.json
+
+  - code: AvoidAdditionalProperties
+    reasons: this is how TypeSpec models records
+    from: vmware.json
+
+  - code: XmsPageableForListCalls
```

Comment:
>XmsPageableForListCalls [](http://example.com/codeflow?start=10&length=23)

I seem to remember one of the reasons to include it is in case ARM injects this into the list response somehow.

Reply:
This suppression has been removed.